lua/entities/gmod_wire_cpu.lua

```diff
@@ -4,8 +4,20 @@ ENT.PrintName       = "Wire ZCPU"
 ENT.Author          = "Black Phoenix"
 ENT.WireDebugName	= "ZCPU"
 
+CreateConVar("wire_cpu_max_frequency", "1400000", {FCVAR_REPLICATED})
+
 if CLIENT then return end -- No more client
 
+cpu_max_frequency = nil
```

Comment:
It'd probably be a lot easier just to set this to the return value of `CreateConVar` and use `GetInt()` directly on line 275.